CMakeLists.txt

```diff
@@ -202,6 +202,10 @@ set(SWIFT_HOST_VARIANT_ARCH "${SWIFT_HOST_VARIANT_ARCH_default}" CACHE STRING
 # This is primarily to support building smaller or faster project files.
 #
 
+# Subsequent options may refer to `StdlibOptions`, which have to be defined first.
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/stdlib/cmake/modules)
+include(StdlibOptions)
```

Comment:
Instead of including `StdlibOptions` here, why don't we move the following into `StdlibOptions`?

```
# Use dispatch as the system scheduler by default.
# For convenience, we set this to false when concurrency is disabled.
set(SWIFT_CONCURRENCY_USES_DISPATCH FALSE)
if(SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY AND "${SWIFT_CONCURRENCY_GLOBAL_EXECUTOR}" STREQUAL "dispatch")
  set(SWIFT_CONCURRENCY_USES_DISPATCH TRUE)
endif()
```

AFAICT it is only used in the stdlib build, so really has no business being set in the root `CMakeLists.txt`

Reply:
^ Was just thinking that.

Reply:
Good point, that's addressed now.